platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/AbstractLongRunningOperation.java

```diff
@@ -120,7 +120,7 @@ public T setJavaHome(File javaHome) {
 
     @Override
     public T setJvmArguments(String... jvmArguments) {
-        operationParamsBuilder.setJvmArguments(rationalizeInput(jvmArguments));
+        operationParamsBuilder.setBaseJvmArguments(CollectionUtils.toList(jvmArguments));
```

Comment:
❓ Shouldn't we keep `rationalizeInput` here?
```suggestion
        operationParamsBuilder.setBaseJvmArguments(rationalizeInput(CollectionUtils.toList(jvmArguments)));
```

Reply:
You are right about this; this can use `rationalizeInput` now.